### ISSUE TITLE
[Keys] keys import is broken, sometimes get stuck in inifite loop

### ISSUE THREAD

**Issue by zramsay** [eris-ltd/eris-cli#697]
still examining edge cases

**Comment by zramsay** [comment]
the problem is deeper than just `data.ImportData()` for example: [this line](https://github.com/eris-ltd/eris-cli/blob/develop/cmd/keys.go#L113) fails to set `do.Source` to the appropriate var, and rather, sets it to "rawgit". This is coming [from init](https://github.com/eris-ltd/eris-cli/blob/develop/cmd/init.go#L30) and is definitely new behaviour. So `eris keys import` is broken. The unit tests didn't catch this because they operate deeper than this problem would present itself.

**Comment by zramsay** [comment]
haven't been able to reproduce since the one time I spotted it. re-naming to relevant issue

**Comment by zramsay** [comment]
getting ride of `eris keys import --src` since key should just be in its standard location. ditto for `eris keys export --dest`. if people want keys in non-standard locations, they can `cp` after the fact.

**Comment by pietv** [comment]
I was experimenting a little with rebuilding containers trying to reproduce this error: I think the error should be similar to the https://github.com/eris-ltd/eris-cli/issues/843 issue with the `eris chains make`. 

The only difference is that the `eris chains make` mounts the `quay.io/eris/data` container image, while `eris keys import` —`quay.io/eris/keys`, so rebuilding the `quay.io/eris/keys` image with the `VOLUME` clause reproduces the issue.

The endless loop for the `eris keys import` is possible when the `data.ImportData()` function is calling itself recursively while failing to create a directory within the data container
```
                _, err := ExecData(doCheck)
                if err != nil {
                        if err := runData(containerName, []string{"mkdir", "-p", do.Destination}
                                return err
                        }
                        return ImportData(do)
                }
```
And that's only possible when there's no `/home/eris/.eris/keys` mount point to write to.

Make sense to add a check for the mount point existence to prevent the loop.

**Comment by zramsay** [comment]
checking mount point leads to import cycle issue (importing the `services` package). fixes in #897 sort this issue (one too many paths joined was the issue)

**Comment by zramsay** [comment]
closed by #897
